code/modules/mob/living/carbon/human/species/human.dm

```diff
@@ -17,4 +17,11 @@
 	worlds tumultous at best."
 
 	reagent_tag = PROCESS_ORG
-	//Has standard darksight of 2.
\ No newline at end of file
+	//Has standard darksight of 2.
+
+/datum/species/human/cat
+	name = "Felinid"
+	name_plural = "Felinids"
+	blurb = "CATGIRLS OMG"
+
+	mutantears = /obj/item/organ/internal/ears/cat
```

Comment:
Would it now make sense to set up Tajaran & Vulpkanin mutantears and get 'em off the head sprites? This also means we may be able to have different selectable ears.

:thinking: